### PR TITLE
fix pipeline: version based on changelog

### PR DESCRIPTION
**Pull request by Nas84**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -272,10 +272,18 @@ jobs:
       - name: Download artifact
         uses: actions/download-artifact@v3
 
-      - name: Get yourloops version from package.json
+      - name: Get yourloops version from changelog
         id: package-version
```
Comment (yanngit) [inline .github/workflows/ci.yml:276]:
```suggestion
        id: changelog-version
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -290,31 +298,20 @@ jobs:
...
       - name: Build and push blip Docker image
         uses: docker/build-push-action@v3
-        env:
-          PR_NUMBER: ${{github.event.issue.number}}
         with:
           context: .
           push: true
           platforms: linux/arm64,linux/amd64
-          tags: ${{ steps.meta.outputs.tags }}
+          tags:  ${{ secrets.REGISTRY }}/blip:${{ steps.package-version.outputs.semver }}
```
Comment (yanngit) [inline .github/workflows/ci.yml:307]: changelog version ?

File: .github/workflows/publish_on_request.yml [hunk 3]
```diff
@@ -67,10 +68,18 @@ jobs:
       - name: Download artifact
         uses: actions/download-artifact@v3
 
-      - name: Get yourloops version from package.json
+      - name: Get yourloops version from changelog
         id: package-version
```
Comment (yanngit) [inline .github/workflows/publish_on_request.yml:72]:
```suggestion
        id: changelog-version
```

File: .github/workflows/publish_on_request.yml [hunk 4]
```diff
@@ -93,10 +102,10 @@ jobs:
           context: .
           push: true
           platforms: linux/arm64,linux/amd64
-          tags:  ${{ secrets.REGISTRY }}/blip:${{ steps.package-version.outputs.semver }}-PR-${{env.PR_NUMBER}}
+          tags:  ${{ secrets.REGISTRY }}/blip:${{ steps.package-version.outputs.semver }}-beta-${{env.PR_NUMBER}}
```
Comment (yanngit) [inline .github/workflows/publish_on_request.yml:105]: changelog version

File: .github/workflows/publish_on_request.yml [hunk 5]
```diff
@@ -93,10 +102,10 @@ jobs:
...
 
       - name: Creating new github tag
         uses: rickstaa/action-create-tag@v1
         with:
-          tag: v${{ steps.package-version.outputs.semver }}
+          tag: v${{ steps.package-version.outputs.semver }}-beta-${{env.PR_NUMBER}}
```
Comment (yanngit) [inline .github/workflows/publish_on_request.yml:110]: changelog version

File: .github/workflows/ci.yml (hunk 2 above)
Comment (yanngit) [reply .github/workflows/ci.yml:307]: Missing beta here ?

File: .github/workflows/ci.yml [hunk 6]
```diff
@@ -290,31 +298,20 @@ jobs:
...
-      # for now disable as we not used the changelog (missing UNRELEASED info)
-  #      - name: Creating new github tag
-  #        uses: rickstaa/action-create-tag@v1
-  #        with:
-  #          tag: ${{ steps.meta.outputs.version }}
-  #          force_push_tag: true
+      - name: Creating new github tag
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: v${{ steps.package-version.outputs.semver }}
```
[the remark this answers is not in the document]
Comment (yanngit) [reply .github/workflows/ci.yml:313]: Missing beta here ?

[the remark this answers is not in the document]
Comment (Nas84) [reply .github/workflows/ci.yml:313]: no because we are on main branch (dblp)

**Review by yanngit — APPROVED** [review]

**Pull request merged**